test/fixtures/dialects/postgres/postgres_pgvector_operators.sql

```diff
@@ -0,0 +1,7 @@
+SELECT * FROM items ORDER BY embedding <-> '[3,1,2]' LIMIT 5;
+
+SELECT * FROM items ORDER BY embedding <=> '[3,1,2]' LIMIT 5;
+
+SELECT * FROM items ORDER BY embedding <+> '[3,1,2]' LIMIT 5;
+
+SELECT * FROM items ORDER BY embedding <#> '[3,1,2]' LIMIT 5;
```

Comment:
nit: newline

Reply:
Resolved in [0e015ef](https://github.com/sqlfluff/sqlfluff/pull/6863/commits/0e015ef0565e24e26a3c906bc9c48a33980ab75d).